plugins/inputs/ipset/ipset_nocompile.go

```diff
@@ -0,0 +1,3 @@
+// +build !linux
```

Comment:
You can remove all of the build flags, it won't hurt to allow this to be built on all platforms.